### PR TITLE
41 feat add google ctf writeups

### PR DESCRIPTION
**Pull request by Stefan-Radu**

### CODE REVIEW

**Comment by Costinteo** [comment]
@Stefan-Radu man you killed it with the writeups god damn. I'll write some this week along with the ones for UIU. Don't forget to paste at least some of those writeups in CTFtime.org as well!

**Comment by Stefan-Radu** [comment]
> Don't forget to paste at least some of those writeups in CTFtime.org as well!

I didn't think of posting on ctftime ups. will do

**Comment by Stefan-Radu** [comment]
is this done? @Costinteo

**Comment by Costinteo** [comment]
> is this done? @Costinteo

Now yes. Theoretically still have Zermatt but I don't really feel like writing it. Wrote those up pretty fast as well. Can you check them out before merging?

File: content/google_ctf_2023/write-flag-where2.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+date: 2023-06-30T13:11:29+03:00
+description: Writeup for Write-Flag-Where2 [Google Ctf 2023]
+author: sunbather
+tags:
+- pwn
+draft: false
+---
+
+## Challenge Description
+This challenge is not a classical pwn
```
[the remark this answers is not in the document]
Comment (Stefan-Radu) [reply content/google_ctf_2023/write-flag-where2.md:12]: @Costinteo only this

[the remark this answers is not in the document]
Comment (Costinteo) [reply content/google_ctf_2023/write-flag-where2.md:12]: Does it actually appear on same line even with header being there?

[the remark this answers is not in the document]
Comment (Costinteo) [reply content/google_ctf_2023/write-flag-where2.md:12]: From what I'm seeing on a live markdown preview online, it seems to not be on the same line.

**Comment by Stefan-Radu** [comment]
Did you run the server locally? I ran it and it appeared on the same line

**Comment by Costinteo** [comment]
okay I commit amended some new lines.

**Pull request merged**